edx_solutions_api_integration/users/views.py

```diff
@@ -357,6 +358,7 @@ def get_queryset(self):
         usernames = css_param_to_list(self.request, 'username')
         organization_display_name = self.request.query_params.get('organization_display_name', None)
         internal_admin_flag = self.request.query_params.get('internal_admin_flag', None)
+        exclude_type = self.request.query_params.get('exclude_type', None)
```

Comment:
Do we send this attribute `exclude_type` for this API?

Reply:
yes, I have made a PR for this https://github.com/mckinseyacademy/mcka_apros/pull/3738/files

Reply:
In the above PR it's sending `exclude_type` to course metrics API `{course_id}/metrics?metrics_required=users_completed,users_passed,avg_grade,avg_progress` while this is Users api `api/users/`?

Reply:
In case of Users api, we are sending `exclude_type ` here https://github.com/mckinseyacademy/mcka_apros/pull/3738/files#diff-1d00614853de0c33b3cb5b834f395a8cR4030 as it will be using through user_api.get_filtered_participants_list(query_params)